docs/topics/control-flow.md

```diff
@@ -143,9 +224,9 @@ when (x) {
 }
 ```
 
-Another option is checking that a value `is` or `!is` of a particular type. Note that,
-due to [smart casts](typecasts.md#smart-casts), you can access the methods and properties of the type without
-any extra checks.
+Additionally, you can check that a value `is` or `!is` a particular type. Note that,
```

Comment:
How about making it consistent with the previous leading line that was super clear:

Additionally, you can check that a value is or isn't of a particular type via the `is` or `!is` keywords: